murdock-worker/murdock-slave-init.sh

```diff
@@ -44,7 +45,9 @@ _start() {
         ${MURDOCK_DOCKER_ARGS} \
         -e CCACHE="ccache" \
         -e CCACHE_MAXSIZE \
+        -e CCACHE_MAXFILES \
```

Comment:
I'm using this locally, I think because of a tweak @kaspar030 shouted over on Matrix.